Celeste.Mod.mm/Patches/TextMenu.cs

```diff
@@ -150,6 +227,39 @@ public class patch_TextMenu : TextMenu {
             return skippedItems;
         }
 
+        /// <inheritdoc cref="TextMenu.Add(TextMenu.Item)"/>
+        [MonoModReplace]
+        public new TextMenu Add(Item item)
+        {
+            items.Add(item);
+            item.Container = this;
+            Add(item.ValueWiggler = Wiggler.Create(0.25f, 3f));
+            Add(item.SelectWiggler = Wiggler.Create(0.25f, 3f));
+            item.ValueWiggler.UseRawDeltaTime = item.SelectWiggler.UseRawDeltaTime = true;
+            if (Selection == -1) {
+                FirstSelection();
+            }
+            if (!BatchMode) {
+                RecalculateSize();
+            }
+            item.Added();
+            return this;
+        }
```

Comment:
This could be changed through MonoModRules to avoid re-implementing, but it may not be worth the effort for something this short.